src/components/Avatar/Avatar.tsx

```diff
@@ -106,19 +112,16 @@ const Avatar: React.FC<AvatarProps> = props => {
         </Typography>
       </Grid>
       {name && (
-        <Typography
```

Comment:
If you wan't to use `h3` variant on `REGULAR` just update it, don't remove the ternary completely.

Reply:
The correct Typography variant for the avatar  is `body1`. The current implementation has `h4` if regular or `h5` if it's small (this small variant doesn't not exist in our design as we already spoke).

So  will update it update the regular avatar (that is the only one we need) to body1 and leave the rest as it was.
`<Typography align='center' component={'h3'}  variant={type === AvatarType.REGULAR ? 'body1' : 'h5'}>`

Reply:
Yes, thanks :+1: Even though, pls next time raise such things on standup or on our Slack channel, before pushing changes that breaks currently working implementation that you previously accepted during code-review. We wouldn't waste some much discussing on that.  From my perspective I did put some effort to create those _extra_ styles for `<Avatar type='small'` />` thinking that we can re-use it in the future.

Reply:
**Simplify Avatar component**
https://digitalrig.atlassian.net/browse/BUD-208

I've created separate task ☝️